astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -2,18 +2,17 @@
 
 import logging
 import typing
+import warnings
 from contextlib import closing
-from datetime import timedelta
 from typing import Any, Callable, List
 
-import requests
 from airflow.exceptions import AirflowException
 
 from snowflake.connector import SnowflakeConnection
 from snowflake.connector.constants import QueryStatus
 
 try:
-    from airflow.providers.snowflake.operators.snowflake import SnowflakeOperator
+    from airflow.providers.snowflake.operators.snowflake import SnowflakeOperator, SnowflakeSqlApiOperator
```

Comment:
I think we'll need to move this SnowflakeSqlApiOperator import outside of this block

Reply:
I think we can remove SnowflakeOperator here. We've update our snowflake provider version